src/Common/tests/hash_table_erase.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iomanip>
+#include <iostream>
+
+#include <Common/HashTable/HashMap.h>
+#include <Common/HashTable/HashSet.h>
+
+
+int main(int, char **)
```

Comment:
BTW, it's better to "unitize" this test (see `gtest_*.cpp` files), so it will be run in CI.

Reply:
Updated.